mobile/lib/models/air_quality_reading.g.dart

```diff
@@ -114,7 +114,7 @@ Site _$SiteFromJson(Map<String, dynamic> json) {
     searchLocation: json['location_name'] as String? ?? '',
     country: json['country'] as String? ?? '',
     region: json['region'] as String? ?? '',
-    source: json['network'] as String? ?? '',
+    dataProvider: json['network'] as String? ?? '',
```

Comment:
Thanks @BenjaminSsempala, did you make these changes yourself, or they were made using build runner??..

Reply:
Thanks @NoahNsimbe
I've regenerated this file using build runner